server/chat-commands/room-settings.ts

```diff
@@ -356,8 +356,8 @@ export const commands: ChatCommands = {
 			buffer += `<code>/permissions view</code></p>`;
 			buffer += `<p><strong>Group permissions:</strong> (will affect multiple commands or part of one command)<br />`;
 			buffer += `<code>` + permissionGroups.join(`</code> <code>`) + `</code></p>`;
-			buffer += `<p><strong>Single-command permissions:</strong> (will affect one command)<br />`;
-			buffer += `<code>` + permissions.join(`</code> <code>`) + `</code></p>`;
+			buffer += `<p><details><summary><strong>Single-command permissions:</strong> (will affect one command)</summary>`;
+			buffer += `<code>` + permissions.join(`</code> <code>`) + `</code></details></p>`;
```

Comment:
:[ why?

Reply:
because it gets REALLY spammy otherwise (thanks, Mafia!)

Reply:
You shouldn't hide all of them just because some of them are spammy.

It might be better to split them apart, with a "sub-command permissions" section.

Reply:
Here, let this get you started:

```
			const permissions = allPermissions.filter(perm => perm.startsWith('/') && !perm.includes(' '));
			const subPermissions = allPermissions.filter(perm => perm.startsWith('/') && perm.includes(' '));
```